config/jobs/gardener/build-gardener-images.yaml

```diff
@@ -0,0 +1,77 @@
+postsubmits:
```

Comment:
I thought about switching to periodic jobs again, but decided to continue using postsubmit jobs
- using periodic jobs we might end up in building the same version many times in phases with only a small number of PRs coming in (e.g. on dev freeze). In other cases we would have to wait for the next periodic job starts until we have the most recent build
- building gardener/gardener takes about 5 minutes only and is scaling quite well horizontally, so we won't run into problems regarding compute times anyhow
- the build app would become more complicated if it was enabled for periodic jobs. They do not submit the commit SHA, so we would have add a `git` binary to the image-builder because [prow git package requires it](https://github.com/kubernetes/test-infra/tree/master/prow/git)
- when looking at the Kubernetes repositories, I found out that they are using [postsubmit jobs for their builds](https://github.com/kubernetes/test-infra/tree/master/config/jobs/image-pushing) too